lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -48,7 +48,6 @@
 )
 from certificates import api as certs_api
 from bulk_email.models import BulkEmailFlag
-from util.date_utils import get_default_time_display
```

Comment:
We should track whether this function is still used after we get through this bash.